corehq/apps/reports/templates/reports/standard/base_template.html

```diff
@@ -24,13 +24,14 @@
     <script src="{% static 'case/js/casexml.js' %}"></script>
     <script src="{% static 'reports/js/tabular.js' %}"></script>
     <script src="{% static 'crud/js/admin.js' %}"></script><!-- Only for MVP indicators -->
+
+    <!-- Scripts that are specific to a single report or interface -->
     <script src="{% static 'data_interfaces/js/case_management.js' %}"></script>
     <script src="{% static 'data_interfaces/js/archive_forms.js' %}"></script>
     <script src="{% static 'hqpillow_retry/js/pillow_errors.js' %}"></script>
+    <script src="{% static 'reports/js/project_health_dashboard.js' %}"></script>
+    <script src="{% static 'reports/js/aggregate_user_status.js' %}"></script>
```

Comment:
it looks like these scripts will now load for every report instead of only for the reports where they were needed.

are we worried about extra bloat that will add to the request, or any unforseen changes these scripts will make to reports where they weren't included before?

Reply:
I'm not worried about unexpected changes, because both scripts do [this kind of thing](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/static/reports/js/project_health_dashboard.js#L147) where their logic only runs when there's an ajax request that matches the report slug.

As for bloat, yes, this will add these scripts to all reports, though they'll be compressed which they weren't previously. I'm not really worried because they're small, but also basically I need to make this change to be able to migrate to requirejs. There'll be opportunity in the future to optimize how requirejs bundles files and which ones are included on which page.